Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM public.ecr.aws/l7o7z1g8/services/registry-config:0.0.22 as registry-config
 
-FROM alpine/helm:3.16.1
+FROM alpine/helm:3.16.2
 
 RUN <<EOF
-    apk update && 
```

Comment:
why remove the `update` command if we are installing the latest version of `yq`? Also if we don't update the package list the `apk upgrade` command below would be redundant right?

Reply:
the  --no-cache option for apk allows packages to be users to install with an index that is updated and used on-the-fly and not cached. so update is redundant.

Upgrade brings the following up to date:
`apk upgrade --no-cache
fetch https://dl-cdn.alpinelinux.org/alpine/v3.20/main/aarch64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.20/community/aarch64/APKINDEX.tar.gz
Upgrading critical system libraries and apk-tools:
(1/1) Upgrading apk-tools (2.14.4-r0 -> 2.14.4-r1)
Executing busybox-1.36.1-r29.trigger
Continuing the upgrade transaction with new apk-tools:
fetch https://dl-cdn.alpinelinux.org/alpine/v3.20/main/aarch64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.20/community/aarch64/APKINDEX.tar.gz
(1/5) Upgrading libcrypto3 (3.3.2-r0 -> 3.3.2-r1)
(2/5) Upgrading libssl3 (3.3.2-r0 -> 3.3.2-r1)
(3/5) Upgrading libcurl (8.10.1-r0 -> 8.11.0-r1)
(4/5) Upgrading curl (8.10.1-r0 -> 8.11.0-r1)
(5/5) Upgrading libexpat (2.6.3-r0 -> 2.6.4-r0)
Executing busybox-1.36.1-r29.trigger
Executing ca-certificates-20240705-r0.trigger
OK: 37 MiB in 34 packages`